packages/core/src/request.ts

```diff
@@ -378,7 +378,18 @@ export class Request<UserData extends Dictionary = Dictionary> {
         this.errorMessages.push(message);
     }
 
-    protected _computeUniqueKey({ url, method, payload, keepUrlFragment, useExtendedUniqueKey }: ComputeUniqueKeyOptions) {
+    // TODO: only for better BC, remove in v4
```

Comment:
Please make an issue to track that :slightly_smiling_face:

Reply:
we have other things in the code that are marked just with a TODO, since they are only about removing stuff (namely the old variants for crawler options like `handlePageFunction`). we can create one issue for all of such things later, for now i'd say a todo in the code is enough.

Reply:
Well, there is a rule in Ruff (Python linter) that forces you to add issue links to TODO comments, and while it's a pain in the backside, it prevents forgotten TODOs :slightly_smiling_face: 

Maybe we could do something similar - it would even make it possible to estimate the difficulty of a "Remove v4 TODOs" issue.